SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -244,7 +244,7 @@ const FString SpatialSessionIdURLOption = TEXT("spatialSessionId=");
 
 const FString LOCATOR_HOST    = TEXT("locator.improbable.io");
 const FString LOCATOR_HOST_CN = TEXT("locator.spatialoschina.com");
-const uint16 LOCATOR_PORT     = 443;
+const uint16  LOCATOR_PORT    = 443;
```

Comment:
Nit : revert spacing change

Reply:
Nah was making it vertical aligned. Looks better.

Reply:
I'm with Nic

Reply:
MIKE GAVE UP

Reply:
Josh ambushed me on a video call and convinced me. Happy with this change.